src/containerapp/azext_containerapp/_params.py

```diff
@@ -306,6 +306,11 @@ def load_arguments(self, _):
         c.argument('managed_env', validator=validate_env_name_or_id, options_list=['--environment'], help="Name or resource ID of the container app's environment.")
         c.argument('environment_type', arg_type=get_enum_type(["managed", "connected"]), help="Type of environment.", is_preview=True)
 
+    # scale
+    with self.argument_context('containerapp', arg_group='Scale') as c:
```

Comment:
could you move to L298. Add this parameter with other scale parameters

Reply:
updated